core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaSchemaRegistry.java

```diff
@@ -988,7 +825,7 @@ private boolean isSubjectVersionDeleted(String subject, int version)
     }
   }
 
-  int getMaxIdInKafkaStore() {
+  public int getMaxIdInKafkaStore() {
```

Comment:
The implementations need both `setMaster` and `getMaxIdInStore`. I would have just used `SchemaRegistry` as the interface, but it's already used and I don't think whereas `SchemaRegistry` looks more like an external API, this new one is all about internal implementation. I'm open to it, just not sure what to call it...